modules/configuring-a-pipeline-server.adoc

```diff
@@ -37,15 +37,15 @@ After the pipeline server is created, the `/metadata` and `/artifacts` folders a
 * Select *Create new data connection* to add a new data connection that your pipeline server can access.
 . If you selected *Create new data connection*, perform the following steps:
 .. In the *Name* field, enter a name for the data connection.
-.. In the *Access key* field, enter your access key ID for Amazon Web Services.
+.. In the *Access key* field, enter the access key ID for your S3-compatible object storage provider.
 .. In the *Secret key* field, enter your secret access key for the account you specified.
```

Comment:
Is there a reason we'd want to keep 'your' here, or would 'the' be more consistent?